content/en/examples/pods/inject/dependent-envars.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dependent-envars-demo
+spec:
+  containers:
+  - name: dependent-envars-demo
+    args:
+    - while true; do echo -en '\n'; echo SERVICE_ADDRESS;printenv SERVICE_ADDRESS;echo UNCHANGE_REFERENCE;printenv UNCHANGE_REFERENCE;echo ESCAPED_REFERENCE;printenv ESCAPED_REFERENCE; sleep 30; done;
```

`UNCHANGE_REFERENCE` should be `UNCHANGED_REFERENCE`, right?
```suggestion
    - while true; do echo -en '\n'; echo SERVICE_ADDRESS; printenv SERVICE_ADDRESS; echo UNCHANGED_REFERENCE; printenv UNCHANGED_REFERENCE; echo ESCAPED_REFERENCE; printenv ESCAPED_REFERENCE; sleep 30; done;
```

You could actually use a single `printf` there, BTW:
```suggestion
    - >
      while sleep 30; do printf "SERVICE_ADDRESS %s; UNCHANGED_REFERENCE %s; ESCAPED_REFERENCE %s\n" "${SERVICE_ADDRESS}" "${UNCHANGED_REFERENCE}" "${ESCAPED_REFERENCE}"; done;
```

:warning: _untested_

---

Done.